pkg/apiserver/devops/devops.go

```diff
@@ -0,0 +1,41 @@
+/*
+
+  Copyright 2019 The KubeSphere Authors.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+ */
+package devops
+
+import (
+	"github.com/emicklei/go-restful"
+	"kubesphere.io/kubesphere/pkg/errors"
+	"kubesphere.io/kubesphere/pkg/models/devops"
+	"net/http"
+)
+
+const HOST = "http://ks-jenkins.kubesphere-devops-system.svc.cluster.local:80"
```

Comment:
Use flag to pass in the address of jenkins, and it is convenient to debug locally.

Reply:
ok,already updated.